### DISCUSSION TITLE
Vercel Github integration fail to link multiple accounts to different repos

### DISCUSSION THREAD

**Discussion by T04435** [vercel/community#5386] [Help]
### Summary

Related to https://github.com/orgs/vercel/discussions/2199

The above mentions the commit author email should be the same one as the Vercel account. Which I have setup with git config on each repo

In my case I have a single github account and multiple Vercel accounts.

Same github account:
repo A => git config => a@domain.com
repo B => git config => b@domain.com

Vercel 2 accounts:
account a => a@domain.com
account b => b@domain.com

The integration breaks when the last PR linked to Vercel is from A and the new PR is B or vice versa.

How can I fix this so I do not have to relink the accounts every time.

### Steps to Reproduce

To reproduce:
- create 2 GitHub repos each one with a different email.
- create 2 Vercel accounts each one with a different email.
- link the repos to vercel

Then when creating PRs on either repo the Vercel integration uses the last linked email not the email from the commit. 

And this message is added. by Vercel bot.

I'm an active member of both Vercel Teams.

<img width="853" alt="Screenshot 2024-01-08 at 1 02 11 pm" src="https://github.com/vercel/community/assets/10654469/0d393183-1035-4188-9ca0-97563468ff29">

**Comment by swarnava** [comment]
Hi there,

It's not possible to link multiple Git account with single Vercel account. You need to create each Vercel account for unique Git accounts tied to different email.

**Reply by T04435** [reply]
@swarnava This is an issue as I think most devs have a single git account and they might have multiple Vercel accounts as we work for different companies.

Would this be considered as a feature to add in the future.

ATM, I have to push one commit to get the error from Vercel click the "request access to the Team" to relink, then every commit after that one works. Until I have to swap to another repo and repeat the process.

Thanks
